test/src/023-reload_safe_path_traversal/main

```diff
@@ -20,13 +20,23 @@ cvmfs_run_test() {
   local reload_cmd
 
   # mount sft repository
-  cvmfs_mount sft.cern.ch \
-    "CVMFS_KCACHE_TIMEOUT=5" \
-    "CVMFS_MAX_RETRIES=3" || return 1
+  if running_on_osx; then
+    cvmfs_mount sft.cern.ch \
+      "CVMFS_KCACHE_TIMEOUT=5" \
+      "CVMFS_MAX_RETRIES=3" || return 11
```

Comment:
The if branch and the else branch are the same except for the return value.  Is this intended?